core/vm/evm.go

```diff
@@ -526,6 +525,33 @@ func (evm *EVM) Create2(caller ContractRef, code []byte, gas uint64, endowment *
 // ChainConfig returns the environment's chain configuration
 func (evm *EVM) ChainConfig() *params.ChainConfig { return evm.chainConfig }
 
+func (evm *EVM) getTobinTax(sender common.Address) (*big.Int, *big.Int, *common.Address, error) {
```

Comment:
This doesn't seem to be using any private EVM methods or properties, I'd avoid adding it as a responsibility to the EVM. Maybe extract `GetTobinTax(*EVM, common.Address)` or `interface TobinTaxGetter { Get(*EVM, common.Address) }`. This probably also makes it unit-testable.

Reply:
Good point, added `getTobinTax(*EVM, common.Address)`